### ISSUE TITLE
accessibilityHazard missing not flagged as EPUB Serious Error

### ISSUE THREAD

**Issue by clapierre** [daisy/ace#206]
Please provide the following details if possible or relevant:

- [v1.0.2] the version of Ace impacted by the issue (as returned by the `ace -v` command).
- [v8.6.0] the version of Node used to run Ace (as returned by the `node -v` command).
- [] your operating system and version.
- [] if the issue is a bug when running Ace: the error message as produced in the log file (you can enable debug logging with the `--verbose` option).
- [] if the issue is about the HTML report: the browser(s) and version(s) you used.
- [] if possible, let us know if you can share a sample input file to help us reproduce the issue.

You flag accessiblitySummary
serious | EPUB | metadata-schema-accessibilitysummaryAce | "undefined"
-- | -- | -- | --

But you don't for accessibilityHazard which is also a required metadata field.
Every conformant EPUB Publication must include the following [schema.org] accessibility metadata:
accessMode, accessiblityFeature, accessibilityHazard, and accessiblitySummary

You flag it as missing but not a serious issue

The following accessibility metadata is missing: schema:accessibilityHazard schema:accessibilitySummary schema:accessibilityAPI schema:accessibilityControl .

(API and Control are optional so that is fine.)

**Comment by danielweck** [comment]
This issue is quite old and the modern versions of Ace actually do report "required" and "recommended" metadata ("serious" and "moderate" severity levels, respectively, to match MUST and SHOULD specs):

https://github.com/daisy/ace/blob/c44670569035e464451e2fb741a9bf1681fb8d95/packages/ace-core/src/checker/checker-epub.js#L75-L76

https://github.com/daisy/ace/blob/c44670569035e464451e2fb741a9bf1681fb8d95/packages/ace-core/src/core/a11y-metadata.js#L62-L192
